samples/tests/import_data_text_entity_extraction_test.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+import os
+
+from uuid import uuid4
+from google.cloud import aiplatform_v1alpha1 as aip
```

Comment:
Is this different from the module used in the samples?   There it was just `from google.cloud import aiplatform`